front/document.form.php

```diff
@@ -135,6 +135,22 @@
         );
     }
     Html::back();
+}  else if (isset($_POST['delete_document'])) {
+    $track = new Ticket();
+    $track->getFromDB((int)$_POST['tickets_id']);
+    $doc = new Document();
+    $doc->getFromDB((int)$_POST['documents_id']);
+    if ($doc->can($doc->getID(), UPDATE)) {
+        $document_item = new Document_Item();
+        $found_document_items = $document_item->find([
+            $track->getAssociatedDocumentsCriteria(),
+            'documents_id' => $doc->getID()
+        ]);
+        foreach ($found_document_items as $item) {
+            $document_item->delete($item, true);
+        }
+    }
+    Html::back();
```

Comment:
This action should probably remain in the `ticket.form.php` script, as it is an action made on a specific ticket.

Reply:
done